docs/modelplugin.md

```diff
@@ -31,9 +31,9 @@ project, and a wrapper **modelmain.go** implementing the ModelPlugin interface.
 They are compiled together with the **go build** command using the
 **-buildmode=plugin** option. 
 
-Many examples of Model Plugins are in the [../modelplugins](../modelplugins)
+Many examples of Model Plugins are in the [modelplugins](../modelplugin)
```

Comment:
I would remove the s even between the parentheses

Reply:
Done.